e2etest/pkg/frr/bgp.go

```diff
@@ -63,3 +63,34 @@ func Routes(exec executor.Executor) (map[string]Route, map[string]Route, error)
 	}
 	return v4Routes, v6Routes, nil
 }
+
+// RawDump dumps all the low leven info as a single string.
+// To be used for debugging in order to print the status of the frr instance.
+func RawDump(exec executor.Executor) (string, error) {
+	res := ""
+	out, err := exec.Exec("vtysh", "-c", "show running-config")
```

Comment:
Will it add value having the command added before its output?
So the result would look something like:
```
show running-config
...
--------
cat /etc/frr/bgpd.conf
...
--------
...
```

Reply:
Yep, even with some kind of separator.